hack/latest-binaries.sh

```diff
@@ -13,8 +12,9 @@ MINOR_VERSION="${1}"
 
 # retrieve the available "VERSION/BUILD_DATE" prefixes (e.g. "1.28.1/2023-09-14")
 # from the binary object keys, sorted in descending semver order, and pick the first one
-LATEST_BINARIES=$(aws s3api list-objects-v2 --bucket amazon-eks --prefix "${MINOR_VERSION}" --query 'Contents[*].[Key]' --output text | cut -d'/' -f-2 | sort -Vru | head -n1)
-
+# If s3api fails; use curl to pull the list. For some reason, reversing order returns nothing, so I dropped that from the sort command
+LATEST_BINARIES=$(aws s3api list-objects-v2 --bucket amazon-eks --prefix "${MINOR_VERSION}" --query 'Contents[*].[Key]' --output text | cut -d'/' -f-2 | sort -Vru | head -n1 || curl -s  "https://amazon-eks.s3.amazonaws.com/?prefix=${MINOR_VERSION}" | xmllint --format  --nocdata - | grep -E  "<Key>${MINOR_VERSION}.*" | sed "s/^.*<Key>//g" | sort -Vu | tail -n 1)
```

Comment:
The sort reversal works for me, what flavor of `sort` do you have?
```suggestion
LATEST_BINARIES=$(aws s3api list-objects-v2 --bucket amazon-eks --prefix "${MINOR_VERSION}" --query 'Contents[*].[Key]' --output text | cut -d'/' -f-2 | sort -Vru | head -n1 || curl -s  "https://amazon-eks.s3.amazonaws.com/?prefix=${MINOR_VERSION}" | xmllint --format  --nocdata - | grep -E  "<Key>${MINOR_VERSION}.*" | sed "s/^.*<Key>//g" | sort -Vru | tail -n 1)
```

Reply:
Completely missed this post. 

If sort works, sort works. I can't remember my build environment from way back when, but it made no sense to my why it wasn't working. 

Therefore, as long as my build environment is the outlier, I have no problem with any changes made.